DFC.ServiceTaxonomy.GraphSync/CosmosDb/CosmosDbDescribeContentItemHelper.cs

```diff
@@ -38,30 +37,20 @@ public class CosmosDbDescribeContentItemHelper : IDescribeContentItemHelper
 
         public async Task<IEnumerable<IQuery<object?>>> GetRelationshipCommands(IDescribeRelationshipsContext context)
         {
-            var currentList = new List<ContentItemRelationship>();
+            var currentList = new List<Tuple<string, string>>();
```

Comment:
I wouldnt use this syntax now - nicer to use the (string, string) with optional names

Reply:
Replaced with shorthand tuple